daniel-robert/server.js

```diff
@@ -106,7 +108,7 @@ app.put('/articles/:id', function(request, response) {
 
 app.delete('/articles/:id', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here... 1)3,4,5; 2)Article.prototype.updateRecord(); 3) Delete
```

Comment:
Not updateRecord(), look in Article.js to see where is jquery delete an article with an specific ID being used.

Reply:
its the deleteRecord() method that interacts with this.